core/src/main/java/com/sequenceiq/cloudbreak/service/datalake/DataLakeStatusCheckerService.java

```diff
@@ -38,8 +39,10 @@ public void validateAvailableState(StackView stack) {
                     .forEach(sdxClusterResponse -> {
                         SdxClusterStatusResponse status = sdxClusterResponse.getStatus();
                         if (!(status.isRollingUpgradeInProgress() || status.isAvailable())) {
-                            throw new BadRequestException("This action requires the Data Lake to be available, but the status is "
-                                    + sdxClusterResponse.getStatusReason());
+                            throw new BadRequestException(String.format("This action requires the Data Lake to be available, " +
+                                    "but the status is '%s', Reason: '%s'.",
+                                    sdxClusterResponse.getStatus().name(),
+                                    Objects.toString(sdxClusterResponse.getStatusReason(), "")));
```

Comment:
Question: could we have `Unknown` as a default in case of `null`?

Reply:
and what if the reason would be omitted only if the response is null?

Reply:
I've already addressed this in my previous comment.